src/main/kotlin/no/nb/bikube/catalogue/collections/repository/CollectionsRepository.kt

```diff
@@ -33,12 +33,14 @@ class CollectionsRepository(
         )
     }
 
-    fun getTitleByName(name: String): Mono<CollectionsModel> {
-        return searchTexts(
+    fun getAllNewspaperTitles(page: Int = 1): Mono<CollectionsModel> {
+        return getRecordsWebClientRequest(
             "record_type=${CollectionsRecordType.WORK} and " +
-            "work.description_type=${CollectionsDescriptionType.SERIAL} and " +
-            "title=\"${name}\""
-        )
+            "work.description_type=${CollectionsDescriptionType.SERIAL}",
```

Comment:
work.description_type kan fjernes. Har gjort det i TT-1515 som ligger til review. Den variablen var for å skille mellom "årsverk" og "serieverk", men nå har vi fjernet årsverk (også på all data som ligger i collections) så alle verk er serieverk.

Eventuelt kan det bare være her, så fjerner jeg når jeg rebaser 1515.